include/plugin_interface/SC_Graph.h

```diff
@@ -35,6 +35,7 @@ struct Graph
 	uint32 mNumControls;
 	float *mControls;
 	float **mMapControls;
+	int32 *mAudioBusOffsets;
```

Comment:
it doesn't break any code, but it does break binaries. you can't use server 3.9 with plugin binaries from 3.8, or vice versa.